Syntaxes/SCSS.sublime-syntax

```diff
@@ -230,12 +231,17 @@ contexts:
         - include: unquoted-string
 
   scss-operators:
-    - match: /|$|%|\*|\-\-|\-|\+\+|\+|~|===|==|=|!=|!==|<=|>=|<<=|>>=|>>>=|<>|<|>|!|&&|\|\||\?\:|%=|\+=|\-=|&=|\bnot\b|\bor\b|\band\b|\bwhen\b|\bfrom\b|\bto\b|\bthrough\b|\bin\b
+    - match: /|==|=|!=|<=|>=|<>|<|>|\-|\+|\*
       scope: keyword.operator.scss
+    - match: (?:\s*)\b(not|or|and|when|from|to|through|in)(?=\s)
+      captures:
+        1: keyword.operator.scss
 
   property-list:
-    - match: '(!#)*\{'
+    - match: '(\s*)\{'
```

Comment:
You may want to change this to `\s*(\{)` and change the `scope` to a `capture` on group 1. Otherwise, you are scoping the white space as `punctuation`.